package.json

```diff
@@ -57,7 +60,7 @@
   },
   "style": "dist/css/bootstrap.css",
   "sass": "scss/bootstrap.scss",
-  "main": "dist/js/bootstrap",
+  "main": "src/js/index",
```

Comment:
I agree with @FezVrasta (and in any case i think you meant `js/src` and not `src/js`)

Nevertheless, there is still some issue with importing specific modules (it happens with either `main` file) . I havn't figured out the root cause, but the symptom is that both `import "bootstrap"` and `import { module } from "bootstrap"` ends up loading **ALL** modules.

To test it i've created a [new branch of an empty scaffold project](https://github.com/IdanCo/webpack-modular/tree/test-bootstrap-webpack), and added @Johann-S [latest commit](https://github.com/twbs/bootstrap/pull/22888/commits/ffca50924eb519fe62bccf30c9d899c12eaf887c) as a dependency. Feel free to clone and do `npm install`. Then `npm run serve` will run the app with the new dependency. **notice** that you should manually fix "package.json" in the bootstrap directory to point to the right file (`dist/js/bootstrap`) before serving.

And on a side note - great job @Johann-S ! With this new build, importing bootstrap with webpack (or any other bundler for that matter) becomes a simple one liner. Sadly it will render the entire installation guide i wrote obsolete, but i'll cope ;-)